manifests/script-lab-local.xml

```diff
@@ -454,6 +464,8 @@
         <bt:Url id="PG.Code.Url" DefaultValue="https://localhost:3000?commands=1" />
         <bt:Url id="PG.Run.Url" DefaultValue="https://localhost:3000/run.html" />
         <bt:Url id="PG.Functions.Url" DefaultValue="https://localhost:3000/functions.html" />
+        <bt:Url id="customfunctionsjs" DefaultValue="https://localhost:3000/customfunctions.js" />
```

Comment:
I think you want this (well, at least the html page -- I still don't get the `js` file concept) on the runner domain instead.  I.e., `localhost:3200` for local and `https://script-lab-runner.azureedge.net` for prod